### PR TITLE
replaced fromCommit and promotionBranch variables to sourceBranch and…

### PR DESCRIPTION
**Pull request by DannyCastroVentura**
… toBranch to mainBranch

# PR details

## What is the purpose of this pull request? (put an "X" next to an item)

- [X] Other, please explain:
- replacing variables

## What changes did you make? (Give an overview)

- replaced config.fromCommit, config.promotionBranch and config.toBranch, to config.sourceBranch and config.mainBranch
- updated the config variable on the other functions as well

### CODE REVIEW

File: copado-function/app/Deploy.js [hunk 1]
```diff
@@ -205,7 +204,7 @@ async function run() {
         Log.info('Merge into promotion branch');
         Log.info('===================');
         Log.info('');
-        Deploy.promote(CONFIG.toBranch, CONFIG.promotionBranch);
+        Deploy.promote(CONFIG.mainBranch, CONFIG.sourceBranch);
```
Comment (JoernBerkefeld) [inline copado-function/app/Deploy.js:207]: promote only takes 1 parameter at  the moment. (this was wrong before...)
pls fix

Comment (JoernBerkefeld) [inline copado-function/app/Deploy.js:207]: promote only takes 1 parameter at the moment. (this was wrong before...)
pls fix

**Review by JoernBerkefeld — CHANGES_REQUESTED** [review]

File: copado-function/app/Deploy.js (hunk 1 above)
Comment (DannyCastroVentura) [reply copado-function/app/Deploy.js]: fixed that, and changed the name of the variables inside the functions (the ones received as parameters) as well, pls check it

Comment (JoernBerkefeld) [reply copado-function/app/Deploy.js]: much appreciated!

**Pull request merged**